deploy/trigger-nightly-update.yaml

```diff
@@ -14,7 +14,7 @@ spec:
           serviceAccountName: pipeline
           containers:
             - name: kuadrant-nightly-update-trigger
-              image: quay.io/curl/curl:latest
+              image: quay.io/kuadrant/testsuite-pipelines-tools:latest
```

Comment:
curl is a lighter image. Can't see the reason to change

Reply:
It is true but the `quay.io/kuadrant/testsuite-pipelines-tools` will already be pulled on the cluster so using another image for `curl` will only raise the storage consumption. (by about 28Mb) I can keep the curl images here but the purpose of using one image for all the pipeline task is to reduce number of images needed to be pulled and therefore the overhead created by them.

Reply:
Changed the curl images back

Reply:
I thought the main reason for using this image is that it contains tooling that we use for our tasks, and if we want to use different tools in a single task, we need the image to contain all of them. I'd be fine with using Alpine images if they had the tools we need. I don't think storage and pulling time are the issues

Reply:
I don't mind using various images either as long as we only maintain one image, all the other images we use in our pipelines should be existing images somebody else is responsible for.